platform/darwin/docs/guides/Style Layers Using Expressions.md.ejs

```diff
@@ -0,0 +1,168 @@
+<%
+  const os = locals.os;
+  const iOS = os === 'iOS';
+  const macOS = os === 'macOS';
+  const cocoaPrefix = iOS ? 'UI' : 'NS';
+  const guide = 'StyleLayersUsingExpressions';
+-%>
+<!--
+  This file is generated.
+  Edit platform/darwin/scripts/generate-style-code.js, then run `make darwin-style-code`.
+-->
+
+# Styling Layers Using Expressions
+
+[Runtime Styling](runtime-styling.html) enables you to modify every aspect of the map’s appearance dynamically as a user interacts with your application. Much of the runtime styling API allows you to specify *non-constant expressions*. A style expression allows users to specify in advance how a layout or paint attribute will vary as the zoom level changes or how the appearance of individual features vary based on metadata provided by a content source.
```

Comment:
There isn’t a concept of a “style expression”; it’s just “expressions” that are used with style layer properties.